docs/user/openstack/etcd-ephemeral-disk.md

```diff
@@ -0,0 +1,203 @@
+# Moving etcd to an ephemeral local disk
+
+You can move etcd from a root volume (Cinder) to a dedicated ephemeral local disk to prevent or resolve performance issues.
+
+## Prerequisites
+
+* An OpenStack cloud with Cinder being functional and enough available storage (at least 75 GB) to accommodate 3 Root Volumes for the OpenShift control plane.
+* An OpenStack cloud whith Nova not using Ceph as a backend for ephemeral disks.
+* OpenShift will be deployed with IPI for now; UPI is not yet documented but technically possible.
+* The MachineConfigPool must match metadata.labels[machineconfiguration.openshift.io/role]. This applies to a controller, worker, or a custom pool.
+* The control-plane machine’s auxiliary storage device, such as /dev/vdb, must match the vdb. Change this reference in all places in the file.
```

Comment:
would it be worth to share commands on how to do that?

Reply:
Yes I sometimes think the same but I also find it painful to maintain the commands which often change over releases. I wonder the same about the amount of details sometimes.

I think the OCP docs should provide more details for the customers, but here in this repo I think it's a best-effort kind of documentation from what I've seen so far so in this case I'm fine if we don't provide that level of details.

Reply:
I'm in fact removing that line which is more confusing than helpful.